app/components/hamburger/hamburger.tsx

```diff
@@ -0,0 +1,182 @@
+          </div>
+          {options.map((option) => (
+            <div
+              key={option.name}
+              className={`mt-2 flex cursor-pointer items-center gap-2 rounded-md p-2 ${
+                selected === option.name ? "bg-white" : "hover:bg-gray-300"
+              }`}
+              onClick={() => handleSelect(option)}
+            >
+              {selected === option.name && (
+                <Check size={20} className="text-grey-800" />
+              )}
+              {option.name}
+            </div>
+          ))}
+        </div>
+      )}
+
+      {error && (
+        <p className="text-red-600">Error switching network: {error.message}</p>
+      )}
+    </div>
+  );
+};
+
+const Hamburger: React.FC<HamburgerProps> = ({ theme }) => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <div className="relative">
+      <button
+        className="z-10 flex h-8 w-8 cursor-pointer flex-col justify-around border-none bg-transparent p-0 focus:outline-none"
+        onClick={() => setOpen(!open)}
+      >
+        <div
+          className={`h-1 w-8 transform rounded bg-gray-800 transition duration-300 ease-in-out ${
+            open ? "translate-y-2 rotate-45" : ""
+          }`}
+        />
+        <div
+          className={`h-1 w-8 rounded bg-gray-800 transition-opacity duration-300 ease-in-out ${
+            open ? "opacity-0" : "opacity-100"
+          }`}
+        />
+        <div
+          className={`h-1 w-8 transform rounded bg-gray-800 transition duration-300 ease-in-out ${
+            open ? "-translate-y-2 -rotate-45" : ""
+          }`}
+        />
+      </button>
+
+      {open && (
+        <div
+          className={`absolute left-0 top-10 ${
+            theme === "dark"
+              ? "bg-base-dark text-white"
+              : "bg-base-light text-black"
+          } flex h-fit w-full flex-col gap-4 rounded-3xl px-[18px] py-[16px] text-center transition-all duration-300 ease-in-out md:h-[316px] md:w-[565px] md:px-[54px] md:py-[40px] md:font-[700]`}
+        >
+          <div
+            className={`flex items-center justify-between gap-[18px] px-1 py-3 ${
+              theme === "dark"
+                ? "border-b border-[#9a9a9a]"
+                : "border-b border-[#9a9a9a]"
+            }`}
+          >
+            <p
+              className={`${
+                theme === "dark" ? "text-grey-500" : "text-grey-500"
+              }`}
+            >
+              Select Network
+            </p>
+            <CustomSelect theme={theme} />
+          </div>
+
+          <button
+            className={`flex w-full items-center justify-center gap-2 py-4 md:h-[60px] md:py-0 md:font-[700] ${
+              theme === "dark"
+                ? "rounded-xl border-[1.75px] border-grey-700 bg-grey-600"
+                : "rounded-xl border border-[#9a9a9a]"
+            } `}
+          >
+            {theme === "dark" ? (
+              <Image
```

Comment:
Use icons from lucide-react instead of images (library big and chevrons)

Reply:
fixed